src/item/StringToItemParser.php

```
@@ -155,6 +155,7 @@ private static function registerBlocks(self $result) : void{
 		$result->registerBlock("banner", fn() => Blocks::BANNER());
 		$result->registerBlock("barrel", fn() => Blocks::BARREL());
 		$result->registerBlock("barrier", fn() => Blocks::BARRIER());
+		$result->registerBlock("structure_void", fn() => Blocks::STRUCTURE_VOID());
```

PHPStan (2.1.15)

Call to an undefined static method pocketmine\block\VanillaBlocks::STRUCTURE_VOID().


(staticMethod.notFound)